react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -1,14 +1,14 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import { getParticipantById } from '../functions';
```

Comment:
Move this import to the in-feature imports section and arrange alphabetically within that section.